### PR TITLE
Revert "No more freeplans by owncube :("

### PR DESCRIPTION
**Pull request by nickvergessen**
Reverts nextcloud/nextcloud.com#1308

### CODE REVIEW

File: assets/providers.json [hunk 1]
```diff
@@ -39,6 +39,20 @@
...
+      },
+	{
+         "title":"OwnCube e.U.",
+         "url":"https:\/\/owncube.com",
+         "flags":[
+            "de",
+            "all"
+         ],
+         "freeplans":true,
+		 "supports":"both",
```
Comment (nickvergessen) [inline assets/providers.json:51]:
```suggestion
         "freeplans":false,
         "supports":"both",
```

**Comment by kesselb** [comment]
Fine by me but it seems the file is unused anyway :)

**Comment by nickvergessen** [comment]
No, it's the partners page. Just not the preferred providers

**Comment by jospoortvliet** [comment]
Ok, I should just remove this file, as I don't think it does anything anymore. The actual data for this is only on the server and has to be manually edited. It isn't in github...

**Comment by jospoortvliet** [comment]
FYI this list is now here: https://github.com/nextcloud/providers

**Comment by jospoortvliet** [comment]
I will ssh into the server and removing ownCube.

**Pull request closed without merging**